extmod/mbedtls_errors/do-test.sh

```diff
@@ -0,0 +1,4 @@
+#! /bin/bash -e
+# Generate mp_mbedtls_errors.c and build the tester app
+./do-mp.sh
+cc -o tester -I../../lib/mbedtls/include/mbedtls/ mp_mbedtls_errors.c tester.c
```

Comment:
I needed to change the include path to `../../lib/mbedtls/include/` to get this script to work.